src/inference/dev_api/openvino/runtime/isync_infer_request.hpp

```diff
@@ -151,12 +151,12 @@ class OPENVINO_RUNTIME_API ISyncInferRequest : public IInferRequest {
                          const std::function<void(ov::SoPtr<ov::ITensor>& tensor)>& allocate_callback);
 
     std::unordered_map<std::shared_ptr<ov::descriptor::Tensor>, std::vector<ov::SoPtr<ov::ITensor>>> m_batched_tensors;
+    ov::SoPtr<ov::ITensor>& get_tensor_ptr(const ov::Output<const ov::Node>& port) const;
```

Comment:
Why do you move this method to protected?

Reply:
It is used in `set_tensors_impl` implementation. Didn't want to duplicate this code, so moved it to protected to use in gpu impl of infer request